src/PMS/PMS5003.cpp

```diff
@@ -136,13 +136,27 @@ int PMS5003::getPm03ParticleCount(void) {
  */
 int PMS5003::getPm01ParticleCount(void) { return pms.getCount1_0(); }
 
+/**
```

Comment:
I think such descriptions don't provide any value, the same information is already included in the method signature. Propose to gradually remove these.